### ISSUE TITLE
Users on Chrome using Windows not able to receive PM requests

### ISSUE THREAD

**Issue by NotAMoose** [euphoria-io/heim#101]
I'm not able to repro this on the same version of Chrome (Version 56.0.2924.87) on a mac, all three users reporting the issue are on PC. They have tried disabling all addons, no dice. 

One user reports being on windows 7, another user says windows 10 (they think).

**Comment by ckingdev** [comment]
Are these new installs of Chrome? Or had they been working previously to receive notifications?

I haven't looked through this yet but an update to Chrome's stable channel got pushed on Thursday, March 9th, that bumped the major version number.

[https://chromereleases.googleblog.com/2017/03/stable-channel-update-for-desktop.html](Google blog post)
[Chromium changelog](https://chromium.googlesource.com/chromium/src/+log/56.0.2924.87..57.0.2987.98?pretty=fuller&n=10000)

**Comment by NotAMoose** [comment]
I don't believe that any of these users had ever received a PM before. I only know they're on `Version 56.0.2924.87` of Chrome, and using various versions of Windows.

**Comment by ckingdev** [comment]
Have they checked that they haven't blacklisted notifications from euphoria, or that they're not disabled completely?

**Comment by jedevc** [comment]
I think I know the reason. The PM is _technically_ received it's just not being shown because of some styling issues. This applies as well to the welcome banner at the top sometimes just being displayed as a green line.

A quick fix I was using is to temporarily resize the screen which for some reason actually displays the messages. Maybe see if that works?

If it's not that, maybe ask them to check the dev console? Maybe there was a rendering error so that the message isn't displayed?

**Comment by jeremyredhead** [comment]
Hopefully #100 get merged soon, but in the meanwhile, one could try installing this fix in user styles manager like [Stylish](https://en.wikipedia.org/wiki/Stylish) or [Stylus](https://github.com/openstyles/stylus#releases):

```css
@-moz-document domain("euphoria.io") {
  .notice-stack {
    position: unset !important;
    left: unset !important;
    right: unset !important;
    height: 0 !important;
  }
}
```

**Comment by CylonicRaider** [comment]
> @-moz-document domain("euphoria.io") {

I wonder if that works in Chrome.

**Comment by jeremyredhead** [comment]
Both extensions I mentioned have an import method which converts such CSS to an internal format. (Even on Firefox, in fact)
